scraper.py

```diff
@@ -0,0 +1,102 @@
+import json
+
+import requests
+import bs4
```

Comment:
We can remove this as it's not used (we're getting everything we need from the import below)